src/main/java/uk/gov/justice/laa/crime/cfecrime/controllers/CfeCrimeController.java

```diff
@@ -32,7 +33,7 @@ public ResponseEntity<CfeCrimeResponse> invoke(@Parameter(
             content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
                     schema = @Schema(implementation = CfeCrimeRequest.class)
             )
-    ) @Valid @RequestBody CfeCrimeRequest request) {
+    ) @Valid @RequestBody CfeCrimeRequest request) throws UndefinedOutcomeException {
```

Comment:
I think the controller should be catching this exception, and returning an error